modules/3d/include/opencv2/3d.hpp

```diff
@@ -8,6 +8,13 @@
 #include "opencv2/core.hpp"
 #include "opencv2/core/types_c.h"
 
+#include "opencv2/3d/depth.hpp"
+#include "opencv2/3d/volume.hpp"
+
+#include "opencv2/3d/detail/pose_graph.hpp"
+#include "opencv2/3d/detail/kinfu_frame.hpp"
+#include "opencv2/3d/detail/submap.hpp"
```

Comment:
Do we need to expose these "detail" headers to users by default?

Reply:
See no other way.
The functions Tests and rgbd module in contrib use these functions; if they are not exposed then they are not exported (using `CV_EXPORTS`) so the linker fails.
At least I hided `submap.hpp` since it's header-only code and a user can include it explicitly.

Reply:
We may export part of available functionality from "3d.hpp". Some specific cases can be included through direct additional includes.

Reply:
Fixed by moving these includes to internal module's `precomp.hpp` file